simulation/model.py

```diff
@@ -207,13 +209,74 @@ def increment_timestep(self):
         self.households_heat_pump_aware_at_current_step = 0
 
 
+def construct_population_awareness(
+    household_population: pd.DataFrame, heat_pump_awareness: float
+) -> List[bool]:
+    """Construct a list assigning heat pump awareness to each agent in the population"""
+    # Randomly assign heat pump awareness to the agent population with a probability given by `heat_pump_awareness`
+    return [
+        random.random() < heat_pump_awareness for _ in range(len(household_population))
+    ]
+
+
+def construct_population_awareness_due_to_campaign(
+    population_heat_pump_awareness: List[bool],
+    heat_pump_awareness_due_to_campaign: float,
+    heat_pump_awareness: float,
```

Comment:
Just to check:

`heat_pump_awareness_due_to_campaign`: the target awareness we're trying to get to with the campaign
`heat_pump_awareness`: the current level of awareness in the absence of the campaign, i.e. baseline awareness?

This function basically tries to select the people who are currently not yet aware, and randomly assign households s.t. after the campaign the total number of 'aware' household = specified target?

If i'm summarising correctly, please could you:
1. Make sure that the doc string correctly summarises what the function is trying to do, including the argument definitions, as it's not very clear.
2. If my understanding is correct, is `heat_pump_awareness` basically the proportion of `population_heat_pump_awareness` that is `True`? If so - can we clarify this in the code as it's a bit unclear.

Reply:
Yes your understanding is correct. I will update the doc strings.